src/test/debuginfo/boxed-struct.rs

```diff
@@ -54,9 +54,9 @@ impl Drop for StructWithDestructor {
 
 fn main() {
 
-    let unique: Box<_> = box StructWithSomePadding { x: 99, y: 999, z: 9999, w: 99999 };
+    let boxed_with_padding: Box<_> = box StructWithSomePadding { x: 99, y: 999, z: 9999, w: 99999 };
 
-    let unique_dtor: Box<_> = box StructWithDestructor { x: 77, y: 777, z: 7777, w: 77777 };
+    let boxed_with_dtor: Box<_> = box StructWithDestructor { x: 77, y: 777, z: 7777, w: 77777 };
```

Comment:
This rename isn't necessary, but was made for consistency. Only `unique` triggered the bug.